hypervisor/arch/x86/guest/vcpuid.c

```diff
@@ -402,6 +402,8 @@ int32_t set_vcpuid_entries(struct acrn_vm *vm)
 			case 0x10U:
 			/* Intel Processor Trace */
 			case 0x14U:
+			/* PCONFIG */
+			case 0x1bU:
```

Comment:
This only skip the set of vcpuid entry. It still has default value inside. How it can stand for disable the feature?

Reply:
We disable them by return the default value 0.